compiler/quilt/nodes.py

```diff
@@ -128,13 +128,14 @@ def _data(self, asa=None):
         store = None
         hash_list = []
         stack = [self]
+        alldfs = True
         while stack:
             node = stack.pop()
             if isinstance(node, GroupNode):
                 stack.extend(child for _, child in sorted(node._items(), reverse=True))
             else:
                 if not isinstance(node._node, core.TableNode):
-                    raise ValueError("Group contains non-dataframe nodes")
+                    alldfs = False
                 if not node._node.hashes:
```

Comment:
You probably need to check for `node._node is None` now, too (same as in the previous PR)